### DISCUSSION TITLE
尝试加入绳网dc服务器，绷不住了

### DISCUSSION THREAD

**Discussion by fashibuding1214** [share121/inter-knot#4566] [灌水]
![Screenshot_2024-09-07-23-07-28-633_com android browser](https://github.com/user-attachments/assets/5b261fdf-6284-4a53-bfe3-d52752e3ffbc)

**Comment by sswznb** [comment]
这种网站别指望用手机自带浏览器能看了

**Comment by Harryhxh** [comment]
要么在谷歌开加速器用，要么就下dc的app用，国内的应该被全ban了

**Reply by fashibuding1214** [reply]
dc我之前倒用过，去谷歌上试试看

**Comment by fashibuding1214** [comment]
进了，进去就是几个大佬在聊代码，我看不懂，但好厉害_(:зゝ∠)_

**Comment by MJSummer** [comment]
唉 米浏览器是这样的 要不是为了用传送门 我巴不得把它冻结掉
